java/libraries/io/src/processing/io/GPIO.java

```diff
@@ -165,7 +165,7 @@ protected static void checkValidPin(int pin) {
    *  the sketch is going to achieve.
    *  @param ms milliseconds to pause
    */ 
-  protected static void delay(int ms) {
+  public static void delay(int ms) {
```

Comment:
Yeah, I'd skip the microseconds version. We're unlikely to get that kind of timing from the device anyway. Easy enough to add later (and part of the philosophy is to add as necessary—never in anticipation).